index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Mourid Portfolio</title>
+</head>
+<body>
+    <section>
+    <nav class="navbar">
+        <div class="container">
+            <div class="logo"><img src="img/My Logo.png" alt=""></img></div>
+        <div class="menu"><img src="img/Union.svg" alt=""></div>
+    </div>
```

Comment:
- [ ] Please read about each tag before using it, as the last code reviewer said you should use the `section` tag in each sectioning content and each section should have an `hx` element, therefore the order of tags here is wrong after removing this section tag which is wrapping all the code you should use the header tag to wrap the toolbar section (logo and the hamburger button), then you should use a section tag to wrap the headline section, (the title, the paragraph and the icons )